packages/kbn-dom-drag-drop/src/sass/drag_drop.scss

```diff
@@ -91,12 +94,12 @@
   }
 }
 
-$lnsLayerPanelDimensionMargin: 8px;
+$reorderItemMargin: $euiSizeS;
 .domDragDrop__reorderableDrop {
   position: absolute;
   width: 100%;
   top: 0;
-  height: calc(100% + #{$lnsLayerPanelDimensionMargin});
+  height: calc(100% + #{$reorderItemMargin / 2});
```

Comment:
```suggestion
  height: calc(100% + #{$euiSizeS / 2});
```

Reply:
Thanks for the suggestion, @MichaelMarcialis! 
Having a named variable was helpful during this redesign work: easier to find what to needs to change if we change size of items. I would like to keep the variable.